bc_obps/registration/migrations/0109_historicalnaicscode_is_regulated_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 5.0.14 on 2025-05-06 15:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('registration', '0108_V3_1_0'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='historicalnaicscode',
+            name='is_regulated',
+            field=models.BooleanField(
+                db_comment='Whether the NAICS code applies to regulated operations', default=True
+            ),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='naicscode',
+            name='is_regulated',
```

Comment:
I left this in because it sounds like we might later have restrictions on this. However, if the reviewer prefers, I can pull it out and we can add it at that later point if we need it